epub34/authoring/index.html

```diff
@@ -6215,6 +6215,22 @@ <h4>Introduction</h4>
 									<p>Some types of screens will render animations and transitions poorly (e.g., those
 										with high latency).</p>
 								</li>
+								<li>
+									<p>
+										Creators should be aware that a reading systems may override some aspects of 
+										the creator's style settings; creators may want to consult the reading systems' 
+										user agent style sheets, if publicly available. Furthermore, reading systems may
+										also modify some rendering aspects as a result of user interaction (e.g., choice 
+										of fonts, text justification, or foreground and background colors).
+									</p>
+								</li>
+								<li>
+									<p>
+										Creators should avoid using [[html]] [^html-global/style^] attributes in content documents. 
```

Comment:
"html attributes" is not as understandable as the previous "inline styles".

Reply:
The second bracketed reference is respec shorthand for the html style attribute, so the text will come out as "[html] style attributes", with "style" linked to the html definition.